src/main/scala/com/github/unchama/seichiassist/subsystems/breakcount/domain/level/SeichiExpAmount.scala

```diff
@@ -15,6 +15,8 @@ case class SeichiExpAmount private (amount: BigDecimal) extends AnyVal {
 
   def add(a: SeichiExpAmount): SeichiExpAmount = mapAmount(_ + a.amount)
 
+  def consume(a: SeichiExpAmount): SeichiExpAmount = mapAmount(_ - a.amount)
```

Comment:
```suggestion
  def subtract(a: SeichiExpAmount): SeichiExpAmount = mapAmount(_ - a.amount)
```

`add`関数の逆のことをやる関数なので、`consume`ではなく`subtract`のほうが良さそうです